### DISCUSSION TITLE
Badges in Navigation Accordion?

### DISCUSSION THREAD

**Discussion by r-this** [nuxt/nuxt#26235] [Questions]
I'm trying to add "New" badges next to links in the navigation accordion of my UI Pro docs, as seen in the attached screenshot. 

I've attempted various configurations in the content's YAML header, such as badge: New, label: New, and nested badge: with Label:, but the badges don't appear. With and without quotes. Etc.

I couldn't find relevant examples in the demo site, github repos, issues, or discussions. What's the correct way to add these badges? Thanks for your help!

![image](https://github.com/nuxt/nuxt/assets/57203638/72546cf5-88f0-41de-82d8-b8aa2c475726)

**Comment by TheAlexLichter** [comment]
Please ask this in https://github.com/nuxt/ui instead ☺️
